extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,147 @@
+
+
+
+## Transparent Metals
+
+Metals effectively absorb all refracted light (light that isn't reflected), preventing transmission.
+The metallic parameter of a glTF material effectively scales the `baseColor` of the material toward black while, at the same time scaling the F0 (reflectivity) value towards 1.0. This makes the material opaque for metallic values of 1.0 because transmitted light is attenuated out by absorption.
+
+## Modeling Absorption
+
+Absorption is usually defined as an amount of light at each frequency that is absorbed over a given distance through a medium (usually described by Beer’s Law). However, since this extension deals exclusively with very thin surfaces, we can treat absorption as a constant. In fact, rather than absorbed light, we can talk about its inverse: transmitted light. The `baseColor` of the material serves this purpose as it already defines how the light that penetrates the surface is colored by the material. In this model, the transmitted light will be modulated by this color as it passes through.
+
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>The baseColor of the material is used to tint the light being transmitted.</em></figcaption>
+</figure>
+
+### Modeling Refraction
+
+Since the surface is considered to be infinitely thin, we will ignore macroscopic refraction caused by the orientation of the surface. However, microfacets on either side of the thin surface will cause light to be refracted in random directions, effectively blurring the transmitted light. This microfacet lobe is exactly the same as the specular lobe except sampled along the line of sight through the surface.
+
+
+## Realtime Implementation Notes
+
+Implementations of transmission are expected to vary, especially in real-time engines, depending on the capabilities of the runtime, but should endeavor to achieve plausibility. 
+
+
+### Refractive Blurring
+Some rasterizer implementations may opt to sample prefiltered IBL or dynamic light probes to simulate refractive blurring due to surface roughness. Others may choose to sample from a blurred version of the rendered scene behind a surface. Raytracers are expected to use more physically correct refraction with a BSDF supporting multi-scattering like the modified [Smith](https://eheitzresearch.wordpress.com/240-2/) model.
+
+<figure>
+  <img src="./figures/Roughness.png"/>
+<figcaption><em>Refraction due to surface roughness.</em></figcaption>
+</figure>
+
+**TODO** - *This is fine for raytracers or for rasterizers that just sample the existing IBL for refracted light. However, what about rasterizers that want to cheaply render transmission in screen-space (I think this is pretty common)? Typically, the refractive blurring is done by sampling mips or manually-blurred versions of the background scene. This can be eye-balled by an implementation but is there a slightly more mathematically-rigorous version that we could recommend?*
```

Comment:
I do think it would be really helpful here to include some best practices on how rasterizers can produce plausible background blurring through rough surfaces.  Are there good references that you can provide here?  What are other rasterizers doing for this?

Reply:
I actually removed all of this. I wonder if it belongs in the spec. or perhaps in a different doc?